src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -307,15 +307,16 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
       case 'EvmCustomTx': {
         if (!inboundAddressData?.router) return undefined
         if (!accountAssetAddress) return undefined
-        const assetAddress = isToken(fromAssetId(assetId).assetReference)
-          ? getAddress(fromAssetId(assetId).assetReference)
-          : zeroAddress
-        const amountOrDustAmountCryptoBaseUnit = // Reuse the savers util as a sane amount for the dust threshold
-          BigInt(
+        const amountOrDustAmountCryptoBaseUnit = (() => {
```

Comment:
we need two values here.

amount for `depositWithExpiry` is the value being sent (eth or token) for the tx

amount for tx and gas estimation should be the eth amount being deposited or dust amount being sent for withdraw

Reply:
This value is correct for `amountCryptoPrecision` on the return args, I am saying we are constructing the memo incorrectly here now as in not passing the desired transfer amount (being zeroed out for tokens)

Reply:
Oh I see what you meant now. Absolutely correct!

Reply:
da29670928268ddb06d858899e2b8d98f441b191 and 4c842ab5acc5160aa6312bf45aa0ed746d56622e

<img width="425" alt="image" src="https://github.com/shapeshift/web/assets/17035424/6e94fd9f-a53f-4c48-b7bf-2e8f2333cbdb">
<img width="688" alt="image" src="https://github.com/shapeshift/web/assets/17035424/e66b1322-aa1c-4613-8698-6b7c5c8dc53e">
<img width="822" alt="image" src="https://github.com/shapeshift/web/assets/17035424/9703c185-4baa-4be9-8e12-01e79535b514">


<img width="463" alt="image" src="https://github.com/shapeshift/web/assets/17035424/fa6ffb05-1d14-4fb0-b346-71aa149da47d">
<img width="692" alt="image" src="https://github.com/shapeshift/web/assets/17035424/e5934ebe-4f45-46c4-aa4e-116f4c85afe3">
<img width="834" alt="image" src="https://github.com/shapeshift/web/assets/17035424/21f89b92-3bc3-4f1d-82eb-6df80681755f">